src/foundations/theme-provider/themes/theme.sidebar.ts

```diff
@@ -0,0 +1,76 @@
+import { blue, indigo, green, red, neutral, orange, teal } from '../../colors';
```

Comment:
Is this theme just applied to the sidebar? I am not sure if this is where we apply the subtheming

Reply:
oh, this is a theme that I created for testing sub-theming ... we will likely not need it in the future or move it to a different place .. I would like to have only 2 default themes in the package, but for now this is fine